doctr/documents/reader.py

```diff
@@ -56,6 +102,8 @@ def read_img(
     # Switch the channel order
     if rgb_output:
         img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+    if orient_img:
+        img = orient_image(img)
```

Comment:
Let's make this an option in the predictor, but it's not a reading feature

Reply:
ok will do